packages/core-dialog/core-dialog.js

```diff
@@ -25,23 +25,37 @@ export default function dialog (dialogs, open) {
     dialog.setAttribute('aria-modal', true)
     dialog.setAttribute('aria-label', options.label || dialog.getAttribute('aria-label'))
     hasBackdrop || dialog.insertAdjacentElement('afterend', document.createElement('backdrop'))
+    options.strict && dialog.setAttribute('data-dialog-strict', options.strict)
```

Comment:
you can set the attribute anyway, and I would suggest storing it in `UUID` instead:
```
dialog.setAttribute(UUID, Boolean(options.strict)) // Make sure it is true/false
```